pkg/config/config.go

```diff
@@ -1076,6 +1076,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("security_agent.expvar_port", 5011)
 	config.BindEnvAndSetDefault("security_agent.log_file", defaultSecurityAgentLogFile)
 	config.BindEnvAndSetDefault("security_agent.remote_tagger", true)
+	config.BindEnvAndSetDefault("security_agent.remote_workloadmeta", false) // TODO(paulcacheux): switch this to true when ready
```

Comment:
```suggestion
	config.BindEnvAndSetDefault("security_agent.remote_workloadmeta", false) // TODO: switch this to true when ready
```

Reply:
I hope to actually remove this comment completely before merging